### PR TITLE
Change depth output to length of pv

### PR DESCRIPTION
**Pull request by Akababa**
Change depth output in UCI to length of PV, for a somewhat more accurate interpretation of "depth".

### CODE REVIEW

**Comment by killerducky** [comment]
There is another PR somewhere or maybe just discussion of keeping track of the maximum depth, I would prefer that to PV depth since as Gary mentioned it should only ever get longer.

File: src/UCTSearch.cpp [hunk 1]
```diff
@@ -245,8 +245,8 @@ void UCTSearch::dump_analysis(int64_t elapsed, bool force_output) {
     float feval = m_root->get_eval(color);
     // UCI-like output wants a depth and a cp, so convert winrate to a cp estimate.
     int cp = 290.680623072 * tan(3.096181612 * (feval - 0.5));
-    // same for nodes to depth, assume nodes = 1.8 ^ depth.
-    int depth = log(float(m_nodes)) / log(1.8);
+    // depth should just be length of pv
+    int depth = std::count(pvstring.begin(), pvstring.end(), ' ') + 1;
```
[the remark this answers is not in the document]
Comment (Akababa) [reply src/UCTSearch.cpp]: Good point; I think the most elegant solution would be to print every time the PV changes. I'll think for a bit on how to best do that!

**Comment by Akababa** [comment]
Thanks for the input, I made the simple change. Even better would be to print every time the PV changes but that's for another PR.

**Comment by killerducky** [comment]
#421 is looking better.

**Pull request closed without merging**